src/libpriv/rpmostree-core.cxx

```diff
@@ -3590,7 +3590,7 @@ apply_rpmfi_overrides (RpmOstreeContext *self,
       if (!S_ISDIR (stbuf.st_mode))
         {
           if (!ostree_break_hardlink (tmprootfs_dfd, fn, FALSE, cancellable, error))
-            return FALSE;
+            return glnx_prefix_error (error, "Copyup %s", fn);
```

Comment:
Self-note: `fn` here and below stands for "filename", stemming from librpm `rpmfiFN`.